services/director-v2/src/simcore_service_director_v2/modules/comp_scheduler/_scheduler_base.py

```diff
@@ -26,6 +26,7 @@
 from models_library.projects_nodes_io import NodeID, NodeIDStr
 from models_library.projects_state import RunningState
 from models_library.services import ServiceType
+from models_library.services_types import RunID
```

Comment:
Ok, so I would really like that this is not just called a RunID.

The `comp_runs` table already contains _runs_ which have an _ID_ as well. I find this very confusing.
I would suggest to call this either:
- ServiceRunID
- or at least import RunID as ServiceRunID

Reply:
Let's go with `ServiceRunID` and make the save in all modules so we avoid confusion.